firmware-nucleo/smart-air-hockey-table/App/driver_display.c

```diff
@@ -141,23 +140,23 @@ void Driver_Display_Print(Display display, uint8_t line, uint8_t offset, const c
 	transmit_end();
 }
 
-void Driver_Display_ShowScore(Display display, uint8_t score_a, uint8_t score_b)
+void Driver_Display_ShowScore(Display display, uint_fast8_t score_a, uint_fast8_t score_b)
 {
 	if (score_a > 99 || score_b > 99) return;
 
-	uint8_t tens_a = (score_a / 10) % 10;
-	uint8_t ones_a = score_a % 10;
-	uint8_t tens_b = (score_b / 10) % 10;
-	uint8_t ones_b = score_b % 10;
+	uint_fast8_t tens_a = (score_a / 10) % 10;
+	uint_fast8_t ones_a = score_a % 10;
+	uint_fast8_t tens_b = (score_b / 10) % 10;
+	uint_fast8_t ones_b = score_b % 10;

```

Comment:
```suggestion
	uint_fast8_t tens_a = (score_a / 10);
	uint_fast8_t ones_a = score_a - tens_a * 10;
	uint_fast8_t tens_b = (score_b / 10);
	uint_fast8_t ones_b = score_b - tens_b * 10;
```

since we are already doing a bunch of optimizations. Since each score will never be greater than 99, we do not need `%10`.

Reply:
actually, amma do this in a separate pr, brb